### PR TITLE
use babel-plugin-lodash to reduce bundle size

### LINKED ISSUE #24: Too big
webpack-bundle-size-analyzer reports that react-shortcuts adds 543.08 KB to my bundle (before compressing):

```
react-shortcuts: 543.08 KB (18.7%)                                                      
  lodash: 526.94 KB (97.0%)
  <self>: 16.14 KB (2.97%)
```

### PR DESCRIPTION
**Pull request by nickaversano**
fixes #24

### CODE REVIEW

**Comment by petrbrzek** [comment]
@nickaversano Thanks for the PR.

File: src/shortcut-manager.js [hunk 1]
```diff
@@ -89,13 +89,8 @@ class ShortcutManager extends EventEmitter {
...
 
-    const shortcuts = _(cursor)
-      .map(this._parseShortcutDescriptor)
-      .flatten()
-      .compact()
-      .value()
-
-    return shortcuts
+    const shortcuts = _.map(cursor, this._parseShortcutDescriptor)
+    return _.values(_.compact(_.flatten(shortcuts)))
```
Comment (petrbrzek) [inline src/shortcut-manager.js]: Do we need `_.values` for anything?

Comment (petrbrzek) [inline src/shortcut-manager.js]: @nickaversano Yeah, here you have to call `value` as the chain of functions are lazily evaluated. So I think we don't need `_.values` in this PR.

**Comment by natew** [comment]
As a side note, what's funny is once you get a big enough application we basically have to manually revert this. Once you hit about 15 of the lodash individual pieces, you'll want to instruct webpack to force sub-modules to use the main lodash bundle rather than all the split ones, otherwise you end up with a bigger overall bundle because lodash split out ones have extra overhead.

**Comment by DimitryDushkin** [comment]
Will it merge?

**Comment by petrbrzek** [comment]
We'll go with #40 instead. But thank you anyway.

**Pull request closed without merging**